qadence/operations.py

```diff
@@ -1085,34 +1086,21 @@ def eigenvalues(self) -> Tensor:
         )
 
 
-# FIXME: better name that stresses difference to `Wait`?
-@dataclass(eq=False, repr=False)
-class AnalogEntanglement(AnalogBlock):
-    parameters: ParamMap = ParamMap(duration=1.0)
-    qubit_support: QubitSupport = QubitSupport("global")
-
-    @property
-    def eigenvalues_generator(self) -> torch.Tensor:
-        return torch.tensor([0.0], dtype=cdouble)
-
-    @property
-    def duration(self) -> Basic:
-        return self.parameters.duration
-
-
 def _cast(T: Any, val: Any) -> Any:
     return val if isinstance(val, T) else T(val)
 
 
-def wait(
+def AnalogInteraction(
```

Comment:
Usually `CamelCase` means that we're dealing with a class. If you want to keep this a function I'd go with `analog_interaction`